README.md

```diff
@@ -55,7 +55,21 @@ The Taichi project was created by [Yuanming Hu (yuanming-hu)](https://github.com
  - [彭于斌 (archibate)](https://github.com/archibate) (OpenGL Compute Shader backend)
  - [Mingkuan Xu (xumingkuan)](https://github.com/xumingkuan) (IR optimization & standardization)
 
-[Kenneth Lozes (KLozes)](https://github.com/KLozes) and [Yu Fang (squarefk)](https://github.com/squarefk) have also made notable contributions. [[List of all contributors]](https://github.com/taichi-dev/taichi/graphs/contributors)
+[Kenneth Lozes (KLozes)](https://github.com/KLozes) and [Yu Fang (squarefk)](https://github.com/squarefk) have also made notable contributions.
+
+[[List of all contributors to Taichi]](https://github.com/taichi-dev/taichi/graphs/contributors)
+
+-------------------------------
+
+The Simplified Chinese documentation (简体中文文档) was created by [Ark (StephenArk30)](https://github.com/StephenArk30). Significant contributions are made by:
+  - [彭于斌 (archibate)](https://github.com/archibate)
+  - [Danni (isdanni)](https://github.com/isdanni)
+  - [Rex (rexwangcc)](https://github.com/rexwangcc)
```

Comment:
```suggestion
  - [Chengchen Wang (rexwangcc)](https://github.com/rexwangcc)
```
Seems like folks are using their full names here, maybe I should do it too...

Reply:
same haha